dialoptions.go

```diff
@@ -446,6 +448,22 @@ func WithDisableServiceConfig() DialOption {
 	})
 }
 
+// WithDefaultServiceConfig returns a DialOption that configures the default service config, which
+// will be used in cases where:
+// 1. WithDisableServiceConfig is called.
+// 2. Resolver does not return service config or if the resolver gets and invalid config.
+func WithDefaultServiceConfig(s string) DialOption {
```

Comment:
As mentioned over GVC, could we add a link or some documentation describing what this string should look like?

Reply:
WIP to export a doc about canonical service config json string.

Reply:
Ack